### PR TITLE
feat(us-autocompletions): add USAutocompletions

### PR DESCRIPTION
**Pull request by kylekwong**
## What
* Adds US Autocompletion resource + test
* Adds US Autocompletion info to the README

## Why
* Customers should be able to use our libraries to make autocompletion requests

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,13 +5,11 @@
...
     "homepage": "https://www.lob.com/",
     "license": "MIT",
-    "authors": [
-        {
-            "name": "Lob",
-            "email": "support@lob.com",
-            "homepage": "https://www.lob.com"
-        }
-    ],
+    "authors": [{
```
Comment (kylekwong) [inline composer.json]: This was done automatically by my IDE formatter; let me know if you'd prefer to keep it as is.

File: tests/Lob/Tests/Resource/USAutocompletionsTest.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<?php
+use Lob\Lob;
```
Comment (dmlittle) [inline tests/Lob/Tests/Resource/USAutocompletionsTest.php]: Nit: can you add a new line before and after the `use` statements?

File: src/Lob/Resource/USAutocompletions.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<?php
+namespace Lob\Resource;
```
Comment (dmlittle) [inline src/Lob/Resource/USAutocompletions.php]: Nit: can you add new lines before, after and in between the `namespace` and `use` imports?

File: src/Lob/Resource/USAutocompletions.php [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    public function autocomplete(array $data)
+    {
+        return $this->sendRequest(
+            'POST',
+            'us_autocompletions',
+            array(),
+            $data
+        );
+    }
+    public function all(array $options = array(), $includeMeta = false)
```
Comment (dmlittle) [inline src/Lob/Resource/USAutocompletions.php]: Nit: can you add new lines between functions definitions?

File: tests/Lob/Tests/Resource/USAutocompletionsTest.php [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    protected function setUp()
+    {
+        $this->lob = new Lob(LOB_TEST_API_KEY);
+        $this->payload = array(
+            'address_prefix' => '185 BER',
+            'city' => 'SAN FRANCISCO',
+            'state' => 'CA',
+        );
+    }
+    public function testVerify()
```
Comment (dmlittle) [inline tests/Lob/Tests/Resource/USAutocompletionsTest.php]: Nit: can you add new lines between function definitions?

File: composer.json (hunk 1 above)
Comment (dmlittle) [reply composer.json]: I prefer this way better 👍

**Pull request merged**